test-apps/letta-ai-sdk-example/src/app/page.tsx

```diff
@@ -26,15 +27,22 @@ async function getAgentId() {
         throw new Error('Missing LETTA_DEFAULT_TEMPLATE_NAME environment variable');
     }
 
-    const response = await lettaCloud.client.templates.agents.create(loadDefaultProject, loadDefaultTemplate)
+    let response;
+    if (process.env.USE_THIS_LOCALLY) {
+        console.log('Using local Letta agent:', loadDefaultTemplate);
+        response = await lettaLocal.client.templates.agents.create(loadDefaultProject, loadDefaultTemplate);
+    } else {
+        console.log('Using Cloud Letta agent:', loadDefaultTemplate);
+        response = await lettaCloud.client.templates.agents.create(loadDefaultProject, loadDefaultTemplate);
+    }
 
     const nextActiveAgentId = response.agents[0].id;
 
     return nextActiveAgentId;
 }
 
 async function getExistingMessages(agentId: string) {
-    return convertToAiSdkMessage(await lettaCloud.client.agents.messages.list(agentId), {allowMessageTypes: ['user_message', 'assistant_message']});
+    return process.env.USE_THIS_LOCALLY ? convertToAiSdkMessage(await lettaLocal.client.agents.messages.list(agentId), {allowMessageTypes: ['user_message', 'assistant_message']}) : convertToAiSdkMessage(await lettaCloud.client.agents.messages.list(agentId), {allowMessageTypes: ['user_message', 'assistant_message']});
```

Comment:
can you put this into a constants file or something

and call it `TEST_MODE` 

then do like

`TEST_MODE === 'local'`